src/scripts/exclusion_term_expansion.py

```diff
@@ -142,7 +143,7 @@ def expand_ontological_exclusions(
     # Variable names: kids0=Children excluded; kids1=Children included
     prefix_sparql_strings = [f'prefix {k}: <{v}>' for k, v in prefix_map.items()]
     df_kids1 = exclusions_df[exclusions_df['exclude_children'] == True]
-    df_kids0 = exclusions_df[exclusions_df['exclude_children'] == False]
+    df_kids0 = exclusions_df[exclusions_df['exclude_children'] != True]
```

Comment:
@matentzn **Ambiguity of null `exclude_children`**
I fixed a bug that was causing me to get no results when I re-ran exclusion table for OMIM. However, I don't know if my fix is correct/preferable, so I opened an issue: https://github.com/monarch-initiative/mondo-ingest/issues/83

Reply:
I was going to make a separate PR for the bugfix, but I just did it here. Probably woulda been best for me to just do it on `main` and rebase, but whatevs.